apicast/src/apicast.lua

```diff
@@ -30,6 +31,41 @@ function _M.init()
   end
 end
 
+function _M.init_worker()
+  local interval = tonumber(os.getenv('AUTO_UPDATE_INTERVAL'), 10)
```

Comment:
I'd extract that 10 to a constant: `DEFAULT_UPDATE_INTERVAL`. Also, we are using 10 as the default interval here and 60 in the `.env` file. I'd rather see the same value in both places for consistency.